packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { concatStyleSets, mergeStyles, Stack } from '@fluentui/react';
+import React, { useRef } from 'react';
+import { GridLayoutStyles } from '..';
+import { useIdentifiers } from '../../identifiers/IdentifierProvider';
+import { BaseCustomStyles } from '../../types';
+import { GridLayout } from '../GridLayout';
+import { HorizontalGalleryStyles } from '../HorizontalGallery';
+import { ResponsiveHorizontalGallery } from '../ResponsiveHorizontalGallery';
+import { HORIZONTAL_GALLERY_BUTTON_WIDTH, HORIZONTAL_GALLERY_GAP } from '../styles/HorizontalGallery.styles';
+import {
+  horizontalGalleryContainerStyle,
+  horizontalGalleryStyle,
+  LARGE_HORIZONTAL_GALLERY_TILE_SIZE_REM,
+  SMALL_HORIZONTAL_GALLERY_TILE_SIZE_REM,
+  videoGalleryContainerStyle,
+  videoGalleryOuterDivStyle
+} from './styles/DefaultLayout.styles';
+import { isNarrowWidth, _useContainerWidth } from '../utils/responsive';
+
+/**
+ * {@link DefaultLayoutStyles} Component Styles.
+ * @public
+ */
+export interface DefaultLayoutStyles extends BaseCustomStyles {
+  /** Styles for the grid layout */
+  gridLayout?: GridLayoutStyles;
+  /** Styles for the horizontal gallery  */
+  horizontalGallery?: HorizontalGalleryStyles;
+}
+
+/**
+ * Props for {@link _DefaultLayout}.
+ *
+ * @private
+ */
+export interface _DefaultLayoutProps {
+  /**
+   * Allows users to pass an object containing custom CSS styles for the gallery container.
+   *
+   * @Example
+   * ```
+   * <_DefaultLayout styles={{ root: { border: 'solid 1px red' } }} />
+   * ```
+   */
+  styles?: DefaultLayoutStyles;
+  /**
+   * Elements to display in the grid
+   */
+  gridElements?: JSX.Element[];
+  /**
+   * Elements to display in the horizontal gallery
+   */
+  horizontalGalleryElements?: JSX.Element[];
+}
+
+/**
+ * DefaultLayout displays elements passed to `gridComponents` prop in the grid area and elements passed to
+ * `horizontalGalleryComponents` prop in the horizontal gallery area.
+ *
+ * @private
+ */
+export const _DefaultLayout = (props: _DefaultLayoutProps): JSX.Element => {
```

Comment:
nit: if this is `private` it doesn't nee the underscores:
```suggestion
export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
```